### ISSUE TITLE
Can't deploy Key using Bluetooth Proxy

### ISSUE THREAD

**Issue by twanjaarsveld** [tesla-local-control/tesla-local-control-addon#123]
is it possible to use a bluetooth proxy ?

![Uploading Screenshot 2024-12-12 222143.png…]()

**Comment by iainbullock** [comment]
No. The Tesla library requires direct hardware control of the Bluetooth device

**Comment by twanjaarsveld** [comment]
Okay thanks, is there any other way to extend the bluetooth range as my HA server is on the other side of my house

**Comment by iainbullock** [comment]
I recommend you install the docker version of the project on a remote device. I use a Raspberry Pi 3b+ in this way, no problems.

https://github.com/tesla-local-control/tesla_ble_mqtt_docker

**Comment by twanjaarsveld** [comment]
That's smart! 
And will this work in combination with the Home Assistant Tesla Fleet integration?

**Comment by iainbullock** [comment]
Yes. Single example you can send commands using Bluetooth and read status using Tesla Fleet API. I'd like to add the ability to read status via Bluetooth but that's a future enhancement

**Comment by twanjaarsveld** [comment]
Awesome, i used a old thinclient with Home Assistant and installed the HA addon and then used my MQTT server on my main HA install 
works perfectly, except the status update like you said but i could combine the entities from the cloud api and this local one
Thanks a lot !

**Comment by twanjaarsveld** [comment]
The "Tesla Fleet API" isn't as simple to use as it was last year, since 2024.1 its become way more difcult to get it to work

**Comment by iainbullock** [comment]
The latest version of this project v0.3.0 can now read status

**Comment by twanjaarsveld** [comment]
Awesome I'll have a look at it

**Comment by twanjaarsveld** [comment]
![Screenshot 2025-01-05 104839](https://github.com/user-attachments/assets/c5520e4f-4bf1-4aa0-980c-92886755c9dc)
All my sensor entities are "unknown"

**Comment by iainbullock** [comment]
Press the 'Force Data Update' button. Currently there is no polling, you have to press it manually or get an automation to do it

**Comment by iainbullock** [comment]
Please put any new issue in a new Issue so others can read it! Thanks

**Comment by twanjaarsveld** [comment]
Thanks this fixed the issue for most entities 
(all the ones i care about)

**Comment by iainbullock** [comment]
Please could you repeat your last comment here so others can benefit from the new Issue, thanks

https://github.com/tesla-local-control/tesla-local-control-addon/issues/131

Which entities still remain as unknown?
